server/grpc/server_test.go

```diff
@@ -5,6 +5,8 @@ package grpc_test
 import (
 	"context"
 	"fmt"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
```

Comment:
Could you sort the imports?

Reply:
[sorted](https://github.com/cosmos/cosmos-sdk/pull/9391/commits/af30c0d375cd65c35e359dbd5305eec93001f5a8)